### DISCUSSION TITLE
Colab Notebook is out of date.

### DISCUSSION THREAD

**Discussion by marcfielding1** [facebookresearch/detectron2#2727] [General]
Heya, is there anywhere I can PR the colab notebook example that's in the docs? Just needs a fix to the version numbers.

The first cell needs torch and torchvision versions set now.

```
# install dependencies: 
!pip install pyyaml==5.1
!pip install torch==1.7
!pip install torchvision==0.8.1
import torch, torchvision
print(torch.__version__, torch.cuda.is_available())
!gcc --version
# opencv is pre-installed on colab
```

**Comment by ppwwyyxx** [comment]
Duplicate of #2711 

The colab notebook doesn't live on github

**Reply by marcfielding1** [reply]
Ahh thanks! Sorry about that, did see it.

**Reply by marcfielding1** [reply]
In actual fact, not quite a duplicate, I was asking how I could contribute to it to fix it? I'm guessing I can't :-)

**Reply by ppwwyyxx** [reply]
It's not on github, so I think you can't
